src/asm/init.S

```diff
@@ -1,38 +1,53 @@
-%define STACK_SIZE 0x400000 ; 4MB
+%include "address.S"
 
+%define STACK_SIZE 0x400000; 4MB FIXME: too large?
+
+extern enable_paging
 extern kernel_entry
 extern load_flat_gdt
 
-section .kernel_stack nobits ; uninitialized section.
-global kernel_stack_init
+section .stack nobits write
```

Comment:
`write` is explicitly mark this section as `writable`.
But in bare-metal binary like here, these flag is unnecessary. Since there is no automatic memory protection.
However, without `write`, sometimes NASM or ld blame us about it...